src/EnergyEvaluation.jl

```diff
@@ -727,13 +943,37 @@ function energy_update!(trial_pos,index,config::Config,potential_variables::ELJP
 
     new_en = dimer_energy_update!(index,dist2_mat,potential_variables.tan_mat,new_dist2_vec,potential_variables.new_tan_vec,en_tot,r_cut,pot)
 
+    return potential_variables,new_en
+
+end
+
+
+function energy_update!(trial_pos,index,config::Config,potential_variables::LookupTableVariables,dist2_mat,new_dist2_vec,en_tot,pot::LookuptablePotential)
+
+    potential_variables.new_tan_vec = [get_tan(trial_pos,b,config.bc) for b in config.pos]
+    potential_variables.new_tan_vec[index] = 0
+
+    new_en = dimer_energy_update!(index,dist2_mat,potential_variables.tan_mat,new_dist2_vec,potential_variables.new_tan_vec,en_tot,pot)
+
     return potential_variables,new_en
 end
+
```

Comment:
It would make sense to combine these two methods that just differ by using rcut or not (same for dimer_enery_update) - is that possible? e.g. by making  rcut a variable with a default value